yt_dlp/extractor/nitter.py

```diff
@@ -168,61 +231,51 @@ def _real_extract(self, url):
+        thumbnail = (
+            self._html_search_meta('og:image', full_webpage, 'thumbnail url')
+            or remove_end('{}{}'.format(base_url, self._html_search_regex(
+                r'<video[^>]+poster="([^"]+)"', webpage, 'thumbnail url', fatal=False)), '%3Asmall'))
 
-        if uploader:
-            title = '%s - %s' % (uploader, title)
-
-        view_count = parse_count(self._html_search_regex(r'<span[^>]+class="icon-play[^>]*></span>\s([^<]+)</div>', webpage, 'view count', fatal=False))
-        like_count = parse_count(self._html_search_regex(r'<span[^>]+class="icon-heart[^>]*></span>\s([^<]+)</div>', webpage, 'like count', fatal=False))
-        repost_count = parse_count(self._html_search_regex(r'<span[^>]+class="icon-retweet[^>]*></span>\s([^<]+)</div>', webpage, 'repost count', fatal=False))
-        comment_count = parse_count(self._html_search_regex(r'<span[^>]+class="icon-comment[^>]*></span>\s([^<]+)</div>', webpage, 'repost count', fatal=False))
-
-        thumbnail = self._html_search_meta('og:image', full_webpage, 'thumbnail url')
-        if not thumbnail:
-            thumbnail = '%s%s' % (base_url, self._html_search_regex(r'<video[^>]+poster="([^"]+)"', webpage, 'thumbnail url', fatal=False))
-            thumbnail = remove_end(thumbnail, '%3Asmall')
-
-        thumbnails = []
-        thumbnail_ids = ('thumb', 'small', 'large', 'medium', 'orig')
-        for id in thumbnail_ids:
-            thumbnails.append({
-                'id': id,
-                'url': thumbnail + '%3A' + id,
-            })
-
-        date = self._html_search_regex(r'<span[^>]+class="tweet-date"[^>]*><a[^>]+title="([^"]+)"', webpage, 'upload date', fatal=False)
-        upload_date = unified_strdate(date)
-        timestamp = unified_timestamp(date)
+        thumbnails = [
+            {'id': id, 'url': f'{thumbnail}%3A{id}'}
+            for id in ('thumb', 'small', 'large', 'medium', 'orig')
+        ]
+
+        date = self._html_search_regex(
+            r'<span[^>]+class="tweet-date"[^>]*><a[^>]+title="([^"]+)"',
+            webpage, 'upload date', default='').replace('·', '')
 
         return {
             'id': video_id,
             'title': title,
             'description': description,
             'uploader': uploader,
-            'timestamp': timestamp,
+            'timestamp': unified_timestamp(date),
             'uploader_id': uploader_id,
-            'uploader_url': uploader_url,
-            'view_count': view_count,
-            'like_count': like_count,
-            'repost_count': repost_count,
-            'comment_count': comment_count,
+            'uploader_url': f'{base_url}/{uploader_id}',
             'formats': formats,
             'thumbnails': thumbnails,
             'thumbnail': thumbnail,
-            'upload_date': upload_date,
-        }
+            'upload_date': unified_strdate(date),
+        } | counts
```

Comment:
```suggestion
            **counts,
        }
```
dict `|` is not in py3.6